pkg/components/master.go

```diff
@@ -159,6 +159,8 @@ func (m *master) createInitScript() string {
 
 	script := []string{
 		initJobWithNativeDriverPrologue(),
+		// TODO: remove || in the future
+		"/usr/bin/yt execute master_exit_read_only '{}' || echo 'master_exit_read_only is supported since 23.2'",
```

Comment:
It will not work. Init job runs only during cluster initialization, and it won't run again during update. You need to create one more job, which will be run after `UpdateStatusWaitingForPodsCreation`.